### ISSUE TITLE
Question for the max_depth used for training.

### ISSUE THREAD

**Issue by pkqbajng** [princeton-vl/OMNI-DC#2]
Hi, thanks for your excellent work. In the training manuscript, you set the max depth to 10. However, in the test manuscript, the value is 300. Does there exists errors in the provided scripts?.

**Comment by pkqbajng** [comment]
Similar issue also exists for the multi_resolution_learnable_gradients_weights in the training and test manuscripts.

**Comment by zuoym15** [comment]
Hi,

Thank you for your interest! These are great questions.

1. During training the depths are normalized so that the median depth is 1, so 10 is a large enough value. During testing, depth is in metric space, and 300 is a safe value larger than the maximum capture range of LiDAR sensors. 
2. I train the model with multi_resolution_learnable_gradients_weights="learnable" and have tested the different options during test time, and "uniform" works the best. I agree that re-train the model with "uniform" sounds more natural, but I just haven't got time to do that. 

Best,
Yiming

**Comment by pkqbajng** [comment]
Thanks for your reply. As the network is scale-invariant, could the median normalization be viewed as a reweighting for the loss computation?

**Comment by zuoym15** [comment]
Ah that's also a perfect point. Yes, since we do another normalization inside the network, the normalization in the dataloader is purely for loss balancing purpose.

**Comment by pkqbajng** [comment]
Thanks for your quick reply!
